package.json

```diff
@@ -76,7 +76,7 @@
   "scripts": {
     "vscode:prepublish": "npm run clean-dist-dir && npm run esbuild-min",
     "clean-dist-dir": "rm -rf dist/",
-    "esbuild-base": "esbuild ./client/extension.js ./server/server.js --bundle --outdir=dist --define:process.env.BASE_DIR='\"dist\"' --external:vscode --format=cjs --platform=node",
+    "esbuild-base": "esbuild ./client/extension.js ./server/server.js --bundle --outdir=dist --define:process.env.BASE_DIR='\"dist\"' --external:vscode --format=cjs --platform=node --target=es2018",
```

Comment:
@igoradamenko ~The Empire~ Crafter did nothing wrong.

I've tried to update some packages and run [vscode-test/sample](https://github.com/microsoft/vscode-test/tree/main/sample) using different versions of Node, but no luck. Then I managed to find out the original crash error.

Here is the trick:

* server part of our extension includes [vscode-languageserver](https://github.com/funbox/vscode-apib-ls/blob/master/server/package.json#L10)
* it includes [vscode-languageserver-protocol](https://github.com/microsoft/vscode-languageserver-node/blob/main/server/package.json#L27)
* it includes [vscode-jsonrpc](https://github.com/microsoft/vscode-languageserver-node/blob/main/protocol/package.json#L21)

vscode-jsonrpc itself includes some relatively fresh ES features like optional chaining: https://github.com/microsoft/vscode-languageserver-node/blob/main/jsonrpc/src/common/linkedMap.ts#L59

So the error was
```
vscode-apib-ls/dist/server/server.js:320
        return this._head?.value;
                          ^
SyntaxError: Unexpected token '.'
```

Since we use relatively old version of VScode in our tests (1.48), I guess that Electron wrapper of that version contains slightly outdated chromium, which does not support optional chaining.

Now, esbuild doc says:

> by default, esbuild's output will take advantage of all modern JS features. For example, a !== void 0 && a !== null ? a : b will become a ?? b when minifying is enabled which makes use of syntax from the [ES2020](https://262.ecma-international.org/11.0/#prod-CoalesceExpression) version of JavaScript.

Since I don't want to force users to use brand new VSCode versions, I set target ES version for esbuild to ES2018, so it can transpile optional chaining and other js features.

As you see, now tests are green =)